Sources/Commandant/Errors.swift

```diff
@@ -9,79 +9,75 @@
 import Foundation
 import Result
 
-#if swift(>=3)
-	public typealias ClientErrorType = ErrorProtocol
-#else
-	public typealias ClientErrorType = ErrorType
-#endif
+public typealias ClientErrorType = Error
```

Comment:
Let's just drop the `typealias`, it doesn't add any value anymore, especially not as `pulibc`.